src/dodal/devices/hutch_shutter.py

```diff
@@ -81,14 +81,14 @@ def __init__(self, prefix: str, name: str = "") -> None:
 
     @AsyncStatus.wrap
     async def set(self, value: ShutterDemand):
-        interlock_state = await self.interlock.shutter_safe_to_operate()
-        if not interlock_state and not TEST_MODE:
-            # If not in test mode, fail. If in test mode, the optics hutch may be open.
-            raise ShutterNotSafeToOperateError(
-                "The hutch has not been locked, not operating shutter."
-            )
         if not TEST_MODE:
             if value == ShutterDemand.OPEN:
+                interlock_state = await self.interlock.shutter_safe_to_operate()
+                if not interlock_state:
+                    # If not in test mode, fail. If in test mode, the optics hutch may be open.
+                    raise ShutterNotSafeToOperateError(
+                        "The hutch has not been locked, not operating shutter."
+                    )
```

Comment:
Is this something that has to be checked in plans or is there a check also done at the EPICS level?